src/features/devtools/components/LayoutInspector/LayoutInspector.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import classes from 'src/features/devtools/components/LayoutInspector/LayoutInspector.module.css';
+import { LayoutInspectorItem } from 'src/features/devtools/components/LayoutInspector/LayoutInspectorItem';
+import { useAppSelector } from 'src/hooks/useAppSelector';
+
+export const LayoutInspector = () => {
+  const { currentView } = useAppSelector((state) => state.formLayout.uiConfig);
+  const layouts = useAppSelector((state) => state.formLayout.layouts);
+
+  const currentLayout = layouts?.[currentView];
```

Comment:
Btw, getting the entire layout hierarchy is as easy as calling `useExprContext()` instead. Using the flat layout in the developer tools dooms us into re-creating the hierarchy in the devtools, so we should continue using the hierarchy instead of the flat layout (and showing the computed properties are much more useful). The `LayoutNode` objects also have methods for getting form data (aka the data model) and validation messages for each component.

Reply:
The thought behind this inspector is more to look at the component declaration and possibly make modifications there (similar to modifying css properties in the chrome dev tools, where computed properties are a separate thing). Then you could play with expressions in the devtools and see the changes happening in real time 😃 

So in my opinion both the raw layout components and computed hierarchy are very useful